runtime/src/lib.rs

```diff
@@ -1456,9 +1456,9 @@ parameter_types! {
 
 #[cfg(all(feature = "private-net", feature = "ready-to-test"))] // order-book
 impl qa_tools::Config for Runtime {
+    type WeightInfo = qa_tools::weights::SubstrateWeight<Runtime>;
     type AssetInfoProvider = Assets;
     type QaToolsWhitelistCapacity = QaToolsWhitelistCapacity;
-    type WeightInfo = qa_tools::weights::SubstrateWeight<Runtime>;
```

Comment:
I don't think we need this change.
Also, `WeightInfo` is placed at the end of configs in most cases.

Reply:
ok, it's intellij that complained about different order within trait and its impl. Thought it makes sense to reorder; I'll fix since we follow it everywhere.